backend/dataall/modules/metadata_forms/db/metadata_form_repository.py

```diff
@@ -140,6 +143,8 @@ def query_entity_metadata_forms(
 
         entity_orgs_uris = entity_orgs_uris or []
         entity_envs_uris = entity_envs_uris or []
+        user_org_uris = user_org_uris or []
+        user_env_uris = user_env_uris or []
```

Comment:
with this change if I am an admin - `user_org_uris` and `user_env_uris` will always be empty List and therefore `orgs` and `envs` will be empty and I will not be able to attach any org or env visibility forms to my resource

While it makes sense that admins are treated special for `query_user_metadata_forms` to see all - I think either:
 - Admins should be treated same as any other user for `query_entity_metadata_forms`
 - OR if admin - `orgs = entity_orgs_uris or []` and `envs = entity_envs_uris or []` ignoring the `user_org_uris` and `user_env_uris`